tardis/visualization/widgets/line_info.py

```diff
@@ -390,8 +393,7 @@ class variable :code:`GROUP_MODES` (default value is
                 )
 
         else:  # species_selected is None
-            # qgrid cannot show empty dataframe properly,
-            # so create one row with empty strings
+            # Create one row with empty strings for empty dataframe
```

Comment:
Can Panel display an empty dataframe?

Reply:
Yes at the start it looks like this- 
<img width="786" height="532" alt="image" src="https://github.com/user-attachments/assets/62896bec-309c-4a72-b2c2-ac368c910fb5" />
Only when the area is selected does it populate the dataframe-
<img width="786" height="532" alt="image" src="https://github.com/user-attachments/assets/2c877f7b-2667-4c44-b080-395b65b3a869" />